lib/repository.js

```diff
@@ -145,7 +145,13 @@ Repository.prototype.getReference = function(name, callback) {
   }, callback);
 };
 
-Repository.getReferences = function(repo, type, refNamesOnly, callback) {
+Repository.getReferences =
+function(repo, type, refNamesOnly, includeSymbolic, callback) {
+  if (typeof callback === "undefined" &&
```

Comment:
Awesome, I'm being picky here as someone could pass a falsy value that isn't undefined and that would break your check.